static/css/section/_desktop.scss

```diff
@@ -1,258 +1,153 @@
-// @section desktop-home
-.row-case-studies {
-  background-color: $dark-aubergine;
-  background-image: url('#{$asset-server}3c1c3b37-background-tile.jpg');
-  color: #fff;
-
-  .pull-quote {
-    margin-bottom: .75em;
-
-    p {
-      color: #fff;
-      font-size: 1em;
-      line-height: 1.5em;
+@charset 'UTF-8';
+
+// ---------------------------------------------------------------
+// ---------------------------------------------------------------
+//
+//  Project:		Ubuntu Front-End Cloud section
+//  Author:			Web Team at Canonical Ltd
+//  Last edited by: Peter Mahnke
```

Comment:
As with https://github.com/ubuntudesign/www.ubuntu.com/pull/870#r79427677, would it be okay to remove "Last edited by" and "Last edited on"?